flixel/ui/FlxButton.hx

```diff
@@ -177,6 +177,8 @@ class FlxTypedButton<T:FlxSprite> extends FlxSprite implements IFlxInput
 	public var pressed(get, never):Bool;
 	public var justPressed(get, never):Bool;
 	
+	private var _labelAsSprite:FlxSprite;
```

Comment:
I think the name ```_spriteLabel:FlxSprite``` would be nicer looking. You might also want to add a comment above this variable declaration explaining why it's necessary.